tests/apps/orderform/test_excel_orderform_parser.py

```diff
@@ -162,99 +70,6 @@ def test_generate_mip_orderform_with_cases(mip_order_parser: ExcelOrderformParse
     assert set(case_obj.panels) == set(["Actionable"])
 
 
-def test_parse_mip_orderform(mip_orderform: str, nr_samples_mip_orderform: int):
-    """Test to parse a mip orderform in xlsx format"""
-    # GIVEN a orderform in excel format
-    assert is_excel(Path(mip_orderform))
-    # GIVEN a orderform API
-    order_form_parser = ExcelOrderformParser()
-    # GIVEN the correct orderform name
-    order_name: str = Path(mip_orderform).stem
-
-    # WHEN parsing the mip orderform
-    order_form_parser.parse_orderform(excel_path=mip_orderform)
-
-    # THEN assert that the correct name was set
-    assert order_form_parser.order_name == order_name
-
-    # THEN assert the number of samples parsed are correct
-    assert len(order_form_parser.samples) == nr_samples_mip_orderform
-
-    # THEN assert that the project type is correct
-    assert order_form_parser.project_type == OrderType.MIP_DNA
-
-
-def test_parse_mip_orderform_no_delivery(
```

Comment:
I think this test does not bring much more value, but introduces another orderform fixture so I removed it. Let me know if you have any objections